tests/lib/snaps/test-snapd-policy-app-consumer/meta/snap.yaml

```diff
@@ -275,6 +275,9 @@ apps:
   netlink-connector:
     command: bin/run
     plugs: [ netlink-connector ]
+  netlink-protocol:
+    command: bin/run
+    plugs: [ netlink-protocol ]
```

Comment:
hmm I think this is the wrong file for gadget provided slots

Reply:
That's entirely true though, see there `raw-volume`. Without it spread test fails. What is then correct place to add the definition for the spread test?

Reply:
I don't think you need it in any file, just add it to the list of gadget provided interfaces in missing_interface_spread_test inside run-checks

Reply:
I fixed this